data/world/immovables/rocks/blackland/init.lua

```diff
@@ -7,7 +7,7 @@ wl.Descriptions():new_immovable_type{
    descname = _ "Rocks 1",
    animation_directory = dirname,
    icon = dirname .. "menu1.png",
-   size = "big",
+   size = "small",
```

Comment:
Same for rocks

Reply:
rocks can be placed just necxt to each other so in fact they are not big immovables requireing big plots

Reply:
Calling the Size of an immovable a Plot Requirement in the help is actually misleading. All Immovables can always be placed on any node, regardless of their size and the node's buildcaps. Workers who create immovables may look specifically for nodes with a given minimum buildcaps though. 
The Size of immovables is taken into account for the computation of the buildcaps and movecaps of nearby fields, and whether the node is passable for bobs. 
For buildings only, the number of nodes they occupy depends on their Size, and buildings are never passable for bobs.

Reply:
ok so maybe rephrase this to size with adding the consequences textually like "can't be walked through by bobs"

Reply:
implemented it, however I am unsure whether a shipconstruction siote is walkable or not.